specification/schemas/Broker.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I think we should omit the `Broker`, `Organization` and `Enterprise` schemas, until we have endpoints that actually return those resources.